nss-day-cohort-29/nutshell-spirit-wind
# ab-loginwork

# Description

Created JS to append Username and Input fields to the DOM. Added Login button with username and password verification functionality. When username && password both match a user in JSON database, runs the function loadUserSpecificPage(). If username && password do not match a user in JSON database, an alert is sent notifying and suggesting registration.


## Type of change




# Testing Instructions for Change Made

1. git fetch --all
2. git checkout ab-loginwork
3. cd into src/lib
4. run grunt
5. in database.json add "users" resource obj with "userName", "password", and "id" keys with values
<img width="330" alt="screen shot 2019-01-10 at 10 26 34 am" src="https://user-images.githubusercontent.com/38840331/50982225-437d3200-14c2-11e9-84ef-d689b948a6f2.png">
6. test that correct username and password combination console.log()'s "This is the user page! 1" and incorrect username/password combination sends an alert
7. user.id should be stored in sessionStorage


# Checklist:
